tests/app/submitter/test_converter.py

```diff
@@ -2,204 +2,329 @@
 
```

Comment:
Maybe I missed it, but now that get_payload is united tested separately. You need a new test here to ensure `data`.  is in the payload. They are tested independently only atm.

Reply:
Added a test 👍